OSDOCS-17079-4: CQA 2.0 for SUPP-1: Remote Health Monitoring (Insight (#114941)
openshift/openshift-docs: modules/removing-filters-from-insights-advisor-recommendations.adoc

You can apply multiple filters to the list of recommendations. When ready, you can remove them individually or completely reset them.

.Procedure

> * To remove filters individually, click the *X* icon next to each filter, including the default filters, to remove them individually.

Reviewer: Two things, your change makes "to remove them individually" redundant, and I _think_ the sentence is more clear if we move the default filters bit. WDYT?
```suggestion
* To remove default or non-default filters individually, click the *X* icon next to each filter.
```

Author: :heavy_check_mark:
